assets/wizards/analytics/views/newspack-custom-events/index.js

```diff
@@ -0,0 +1,118 @@
+/* global newspack_analytics_wizard_data */
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	Button,
+	Grid,
+	Notice,
+	SectionHeader,
+	TextControl,
+	withWizardScreen,
+} from '../../../../components/src';
+
+/**
+ * Analytics Custom Events screen.
+ */
+class NewspackCustomEvents extends Component {
+	state = {
+		ga4Credendials: newspack_analytics_wizard_data.ga4_credentials,
+		error: false,
+	};
+
+	handleAPIError = ( { message: error } ) => this.setState( { error } );
+
+	updateGa4Credentials = () => {
+		const { wizardApiFetch } = this.props;
+		wizardApiFetch( {
+			path: '/newspack/v1/wizard/analytics/ga4-credentials',
+			method: 'POST',
+			data: {
+				measurement_id: this.state.ga4Credendials.measurement_id,
+				measurement_protocol_secret: this.state.ga4Credendials.measurement_protocol_secret,
```

Comment:
Our UI design pattern uses a "quiet" loading animation when updating a settings page (and the regular loading pattern when the entire application is loading).

```suggestion
				measurement_protocol_secret: this.state.ga4Credendials.measurement_protocol_secret,
				quiet: true,
```